test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPackCommandTest.cs

```diff
@@ -2321,6 +2327,12 @@ public void PackCommand_InstallPackageToOutputPathWithResponseFile()
                     Assert.Equal("the description", nuspecReader.GetDescription());
                     Assert.Equal("Copyright (C) Microsoft 2013", nuspecReader.GetCopyright());
                     Assert.Equal("Microsoft,Sample,CustomTag", nuspecReader.GetTags());
+                    // Validate the nuspec round trips.
+                    using (var nuspecStream = nupkgReader.GetStream($"packageA.nuspec"))
+                    {
+                        var manifest = Packaging.Manifest.ReadFrom(nuspecStream, true);
+
+                    }
```

Comment:
These blocks should assert (or do something) with `manifest`. The comment says it validates something, but all it does is make sure it doesn't throw an exception

Reply:
I'll assert that the result is not null. 

The expectation is that it doesn't throw really.